src/components/header/_macro.njk

```diff
@@ -32,43 +32,67 @@
+                            </div>
+                            <div class="ons-header__org-logo ons-header__org-logo--small">
+                                {% if params.mastheadLogo.small %}
+                                    {{ params.mastheadLogo.small | safe }}
+                                {% elif params.mastheadLogo.large %}
+                                    {{ params.mastheadLogo.large | safe }}
+                                {% else %}
+                                    {{-
+                                        onsIcon({
+                                         "iconType": 'ons-logo-stacked-' + currentLanguageISOCode,
+                                            "altText": 'Office for National Statistics logo'
+                                        })
+                                    -}}
+                                {% endif %}
+                            </div>
+                            {% if params.mastheadLogoUrl %}
+                                </a>
                             {% endif %}
-                        </div>
-                        {% if params.mastheadLogoUrl %}
-                            </a>
-                        {% endif %}
+                        {% else %}
+                            <div class="ons-header__org-logo ons-header__org-logo--multi">
+                                {% set logos = [params.mastheadLogo.multipleLogos.logo1, params.mastheadLogo.multipleLogos.logo2, params.mastheadLogo.multipleLogos.logo3] %}
+                                {% for logo in logos %}
+                                    {% if logo.logoURL %}
+                                        <a class="ons-header__org-logo-link" href="{{ logo.logoURL }}">
+                                    {% endif %}
+                                    {% if logo.logoImage != "ONS Logo" %}
+                                        {{ logo.logoImage | safe }}
+                                    {% else %}
+                                        {{-
+                                            onsIcon({
+                                                "iconType": 'ons-logo-stacked-' + currentLanguageISOCode,
+                                                "altText": 'Office for National Statistics logo'
+                                            })
+                                        -}}
+                                    {% endif %}
+                                    {% if logo.logoURL %}
+                                        </a>
+                                    {% endif %}
+                                {% endfor %}
+                            </div>
+                        {% endif -%}
                     </div>
                     {% if params.language or params.serviceLinks %}
-                        <div class="ons-header__links ons-grid__col ons-col-auto">
+                        <div class="ons-header__links ons-grid__col ons-col-auto ons-u-ml-auto">
```

Comment:
```suggestion
                        <div class="ons-header__links ons-grid__col ons-u-ml-auto">
```
Think this class can be removed doesn't seem to be doing anything as auto is already being set